recipe/meta.yaml

```diff
@@ -10,32 +10,44 @@ source:
   sha256: dd5be4a72d8a2995c3f583cf82bf3cd1a9544cfdabf2d22595b67aff07349666
 
 build:
-  noarch: python
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  number: 1
+  number: 0
+  skip: True  # [py<36]
 
 requirements:
   host:
-    - python >=3.6
-    - jsonpath
+    - python
     - pip
+    - setuptools
+    - wheel
   run:
-    - python >=3.6
+    - python
 
 test:
+  source_files:
+    - test
   imports:
     - jsonpath
-  commands:
-    - pip check
   requires:
     - pip
+    - pytest
+  commands:
+    - pip check
+    - pytest -v test
 
 about:
   home: https://github.com/zhangxianbing/jsonpath-python
   summary: A more powerful JSONPath implementation in modern python
+  description: |
+    A more powerful JSONPath implementation in modern python.
   license: MIT
   license_file: LICENSE
+  license_family: MIT
+  doc_url: https://github.com/zhangxianbing/jsonpath-python
+  dev_url: https://github.com/zhangxianbing/jsonpath-python
 
 extra:
   recipe-maintainers:
     - jan-janssen
+  skip-lints:
+    - host_section_needs_exact_pinnings
```

Comment:
```suggestion
```
This is not needed anymore. I would also recommend to not add these skips in the future. I prefer to have a false alarm than not being warned that something is broken.